tests/qgis-projects/tests/test_cache.md

```diff
@@ -0,0 +1,13 @@
+This test is to check the cache.
+The test could be:
+- [ ] Config the tile cache as file cache
+- [ ] Create the the cache with the lizmap command line
```

Comment:
@drillinP Please give the command line to avoid testers to search for it.
Layer is displayed as EPSG:3857 because you put OSM as base layers, did you test with cache generated with this projection and not EPSG:2154?

Reply:
It is with EPSG:3857 that I mainly did the test